### PR TITLE
Nicer formatting of ember aliases in README

### PR DESCRIPTION
**Pull request by ultimatemonty**
NIcer formatting for the list of aliases

### CODE REVIEW

File: plugins/ember-cli/README.md [hunk 1]
```diff
@@ -1,22 +1,24 @@
 # Ember-cli
 
-**Maintainer:** [BilalBudhani](http://www.github.com/BilalBudhani)
+**Maintainer:*[BilalBudhani](http://www.github.com/BilalBudhani)
```
Comment (mcornella) [inline plugins/ember-cli/README.md]: :+1: but revert this line's change, otherwise the formatting looks weird.

Comment (ultimatemonty) [inline plugins/ember-cli/README.md]: @mcornella just to confirm you want me to remove the maintainer line?

Comment (mcornella) [inline plugins/ember-cli/README.md]: Keep the current asterisk, _i.e._ don't change the line. This is how it should look:
capture20150903150442226
This is how it looks with your change:
capture20150903150457561

Comment (ultimatemonty) [inline plugins/ember-cli/README.md]: gotcha

**Comment by ultimatemonty** [comment]
@mcornella done. sorry about that

**Comment by mcornella** [comment]
Oh bummer, it seems the `ember-cli` plugin was [updated 3 months ago](https://github.com/robbyrussell/oh-my-zsh/pull/3989), can you rebase your PR? Do you know how to do it?

**Comment by ultimatemonty** [comment]
yeah I'll rebase in a bit.

**Comment by ultimatemonty** [comment]
@mcornella sorry this took me so long to get back to. Rebase was being super annoying so I just used `merge --no-ff`. Let me know if this works.

**Comment by mcornella** [comment]
Sorry, lost track of this. :+1:

**Pull request merged**